lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -0,0 +1,299 @@
+<%page args="section_data" expression_filter="h"/>
+<%namespace name='static' file='/static_content.html'/>
+<%!
+from django.utils.translation import ugettext as _
+from django.utils.translation import pgettext
+from openedx.core.djangolib.markup import HTML, Text
+%>
+<fieldset class="batch-enrollment membership-section">
+    <legend id="heading-batch-enrollment" class="hd hd-3">${_("Batch Enrollment")}</legend>
+    % if not section_data['access']['admin']:
+    <label>
+      ${_("Batch Enrollment is disabled. Please ask users to enroll via ")}
+      <a href="${settings.MARKETING_SITE_BASE_URL}">MITx Online</a>
+      ${_(" or contact ")}
+      <a href="mailto:mitx-et@mit.edu">mitx-et@mit.edu</a>
```

Comment:
A couple of thoughts I wanted to bring up:

- I haven't seen this address before, We've used https://github.com/mitodl/mitxonline/blob/main/frontend/public/src/components/ReceiptPageDetailCard.js#L59 in MITxOnline though.
- Thought this could be a configurable value as well. I see [in `ol-infrastructure`](https://github.com/mitodl/ol-infrastructure/blame/f3264070d7b197b15ada2484cd249148ff70a47c/src/bilder/images/edxapp/templates/edxapp/mitxonline/common_values.yml.tmpl#L406)  but this is very old. I think we can go with this hard-coded address if we're sure that it's not going to change any time sooner or more frequently.

FYI, @pdpinch

Reply:
> I think we can go with this hard-coded address if we're sure that it's not going to change any time sooner or more frequently.

I agree. 

mitx-et@mit.edu is a support address for course faculty.